### PR TITLE
add UriRedirect result class and tests

### PR DESCRIPTION
**Pull request by sbatschelet**
* UriRedirect schema credit @racke

### CODE REVIEW

File: lib/Interchange6/Schema/Result/UriRedirect.pm [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+};
+
+=head2 last_used
+
+Date and time when this record was last used returned as L<DateTime> object.
+Value is auto-set on insert and update.
+
+=cut
+
+column last_used => {
```
Comment (SysPete) [inline lib/Interchange6/Schema/Result/UriRedirect.pm]: last_used? not sure about this. Can you explain where.how this is used?

File: lib/Interchange6/Schema/Result/UriRedirect.pm [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+=head2 status_code
+
+Http status code passed during redirect.
+Default 301 "Moved Permanently".
+
+=cut
+
+column status_code => {
+    data_type         => "integer",
```
Comment (SysPete) [inline lib/Interchange6/Schema/Result/UriRedirect.pm]: may as well be smallint - saves a few bytes

File: lib/Interchange6/Schema/Result/UriRedirect.pm (hunk 1 above)
Comment (racke) [inline lib/Interchange6/Schema/Result/UriRedirect.pm]: On 06/30/2015 03:39 PM, Peter Mottram wrote:
> In lib/Interchange6/Schema/Result/UriRedirect.pm <https://github.com/interchange/interchange6-schema/pull/180#discussion_r33572961>:
> 
>> +
>> +=cut
>> +
>> +column created => {
>> +    data_type         => "datetime",
>> +    set_on_create     => 1
>> +};
>> +
>> +=head2 last_used
>> +
>> +Date and time when this record was last used returned as L<DateTime> object.
>> +Value is auto-set on insert and update.
>> +
>> +=cut
>> +
>> +column last_used => {
> 
> last_used? not sure about this. Can you explain where.how this is used?

To purge redirects which are obsolete, that was the idea. But in our case it should
be datetime. It is also a performance hit, so maybe it should be update by logs
not from DPIC6.

Regards
         Racke


-- 
Perl and Dancer Development

Visit our Perl::Dancer conference 2015.

**Comment by sbatschelet** [comment]
thank you gentlemen

**Pull request merged**